google-ads-examples/src/main/java/com/google/ads/googleads/examples/campaignmanagement/SetBidModifier.java

```diff
@@ -0,0 +1,135 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.ads.googleads.examples.campaignmanagement;
+
+import com.beust.jcommander.Parameter;
+import com.google.ads.googleads.examples.utils.ArgumentNames;
+import com.google.ads.googleads.examples.utils.CodeSampleParams;
+import com.google.ads.googleads.lib.GoogleAdsClient;
+import com.google.ads.googleads.lib.utils.FieldMasks;
+import com.google.ads.googleads.v2.common.DeviceInfo;
+import com.google.ads.googleads.v2.enums.DeviceEnum.Device;
+import com.google.ads.googleads.v2.services.CampaignCriterionOperation;
+import com.google.ads.googleads.v2.errors.GoogleAdsError;
+import com.google.ads.googleads.v2.errors.GoogleAdsException;
+import com.google.ads.googleads.v2.resources.CampaignCriterion;
+import com.google.ads.googleads.v2.services.CampaignCriterionServiceClient;
+import com.google.ads.googleads.v2.services.MutateCampaignCriteriaResponse;
+import com.google.ads.googleads.v2.services.MutateCampaignCriterionResult;
+import com.google.ads.googleads.v2.utils.ResourceNames;
+import com.google.common.collect.ImmutableList;
+import com.google.protobuf.FloatValue;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+
+/** Sets a bid modifier for the mobile platform on given campaign. */
+public class SetBidModifier {
```

Comment:
Per discussion offline, let's name this `UpdateCampaignCriterionBidModifier` to:
1. Use the term 'update' to match the API (vs. 'set' in the AdWords API).
2. Make it clear which type of object the example updates.